www/gatsby-node.js

```diff
@@ -462,14 +462,25 @@ exports.onCreateNode = ({ node, actions, getNode, reporter }) => {
         slug = `/${parsedFilePath.dir}/`
       }
 
-      // Set released status for blog posts.
+      // Set released status and `published at` for blog posts.
       if (_.includes(parsedFilePath.dir, `blog`)) {
         let released = false
         const date = _.get(node, `frontmatter.date`)
         if (date) {
           released = moment().isSameOrAfter(moment.utc(date))
         }
         createNodeField({ node, name: `released`, value: released })
+
+        const canonicalLink = _.get(node, `frontmatter.canonicalLink`)
+        let publishedAt = _.get(node, `frontmatter.publishedAt`)
```

Comment:
This can be a `const`. Other than that this looks good to me :+1:
Thanks for implementing the feedback so quickly!

@pieh do you have anything else?

Reply:
Oops, totally missed that. Thanks.